spec/support/setup_buckets.rb

```diff
@@ -0,0 +1,18 @@
+require 'aws-sdk-s3'
+
+RSpec.configure do |c|
+  c.before(:suite) do
+    minio_config = {
```

Comment:
Why minio_config ?  Its just S3 ?  we might change from minio one day and your code shouldnt really care.

Picky I know - just think our test suite shouldnt really care what S3 server it is using

Reply:
Good idea - fixed in 8c78e49